tests/client/amo/containers/TestApp.js

```diff
@@ -10,19 +10,29 @@ import * as api from 'core/api';
 import { getFakeI18nInst } from 'tests/client/helpers';
 
 
+class FakeMastHeadComponent extends React.Component {
+  render() {
+    // eslint-disable-next-line react/prop-types
+    return <div>{this.props.children}</div>;
+  }
+}
+
+
 describe('App', () => {
   it('renders its children', () => {
+    // eslint-disable-next-line react/no-multi-comp
     class MyComponent extends React.Component {
       render() {
         return <p>The component</p>;
       }
     }
     const i18n = getFakeI18nInst();
     const root = renderIntoDocument(
-      <AppBase i18n={i18n} isAuthenticated>
+      <AppBase i18n={i18n} isAuthenticated
+        MastHeadComponent={FakeMastHeadComponent}>
```

Comment:
Most of the code seems to favour lining up attributes instead of newline + indenting them, but I'll admit it's inconsistent.

Mostly pointing that out... we should be consistent eventually but do what you like, I mildly prefer attrs-lining-up but it's no biggie either way to me.